models/sushiswap/bnb/sushiswap_bnb_sources.yml

```diff
@@ -0,0 +1,67 @@
+version: 2
+
+sources:
+  - name: sushi_bnb
+    description: "Bnb decoded tables related to Sushiswap contract"
+    freshness:
+      warn_after: { count: 12, period: hour }
+    tables:
+      - name: UniswapV2Router02_call_swapETHForExactTokens
+      - name: UniswapV2Router02_call_swapExactETHForTokens
+      - name: UniswapV2Router02_call_swapExactETHForTokensSupportingFeeOnTransferTokens
+      - name: UniswapV2Router02_call_swapExactTokensForETH
+      - name: UniswapV2Router02_call_swapExactTokensForETHSupportingFeeOnTransferTokens
+      - name: UniswapV2Router02_call_swapExactTokensForTokens
+      - name: UniswapV2Router02_call_swapExactTokensForTokensSupportingFeeOnTransferTokens
+      - name: UniswapV2Router02_call_swapTokensForExactETH
+      - name: UniswapV2Router02_call_swapTokensForExactTokens
+      - name: UniswapV2Pair_evt_Swap
```

Comment:
cc. @jeff-dude 
There may be multiple addresses/instances as `UniswapV2Pair` on chain.
We may need to decode and `union all` all these contracts' evts at the same time.

Reply:
cool, i'll work on it this weekend

Reply:
feel free to tag me next week if you guys would like me to look 👍  thanks for the patience and continual builds!